2018 Robot/src/org/usfirst/frc/team4999/robot/OI.java

```diff
@@ -44,4 +47,19 @@
 	// until it is finished as determined by it's isFinished method.
 	// button.whenReleased(new ExampleCommand());
 	
+	Trigger killPID = new KillPID();
+	Trigger driveOvercurrent = new DriveOvercurrent();
+	Trigger liftOvercurrent = new LiftOvercurrent();
+	Trigger intake = new Intake();
+	Trigger outtake = new Outtake();
+	
+	public OI() {
+		killPID.whenActive(new TeleopNoPID());
+		driveOvercurrent.whenActive(new KillDrive());
+		liftOvercurrent.whenActive(new KillLift());
+		
+		intake.whenActive(new Grab());
+		
+		outtake.whileActive(new Release());
```

Comment:
Alright, I mapped it so that one button either intakes or shoots (depending on whether a cube is currently being held) and another button toggles the arms.